tools/ci_build/build.py

```diff
@@ -154,6 +154,7 @@ def parse_arguments():
     parser.add_argument("--use_full_protobuf", action='store_true', help="Use the full protobuf library")
     parser.add_argument("--disable_contrib_ops", action='store_true', help="Disable contrib ops (reduces binary size)")
     parser.add_argument("--skip_onnx_tests", action='store_true', help="Explicitly disable all onnx related tests")
+    parser.add_argument("--skip_winml_tests", action='store_true', help="Explicitly disable all WinML related tests")
```

Comment:
>skip_winml_tests [](start = 27, length = 16)

do we want these to be sepereate?  what are onnx tests vs. winml tests?

Reply:
We could do a single switch to disable all tests, I'm not strongly inclined to either options. What do you prefer?